### DISCUSSION TITLE
Styling table filter menu

### DISCUSSION THREAD

**Discussion by MelvinBriant** [primefaces/community#2535] [PrimeNG]
Hello everybody,

I've question about filter menu, i'm trying to make some changes in CSS but nothing works. I tried using class name with ng-deep.
I tried to implement it on my main CSS file but also on my specific project CSS file.
Here what i would like to change : 
![filter-menu](https://github.com/user-attachments/assets/bd10712d-01a0-4d8b-9067-6406f4125eb1)

I'm using PrimeNG 16.5.1

Thank's in advance for your response.

**Comment by qwadrox** [comment]
@MelvinBriant 

About theming there is a section in the documentation: https://primeng.org/theming
In version 16 the best way to style your components is with the [primeng-sass-theme](https://github.com/primefaces/primeng-sass-theme)

You can find videos on yt how to use that: https://www.youtube.com/watch?v=JAapJf6q1cw
